## Ejercicio alturas 
De las 3 hipótesis planteadas en la presentación, generar la información que permita obtener los parámetros estadísticos que nos servirán para rechazar o no las hipótesis nulas planteadas.

+10 Para obtener estos puntos será necesario investigar un estadístico de prueba, y presentarlo en clase, tanto la metodología como la aplicación de esta a los datos mencionados anteriormente, y deberán de explicar como analizaron estos datos y si se rechazo o no las hipótesis correspondientes.

De 3 variables diferentes de la BD utilizada para el examen del primer parcial, plantear 3 hipótesis, y generar los parámetros estadísticos que sean necesarios para validar las hipótesis nulas, con los estadísticos de prueba que veremos la próxima clase.


Definir Hipótesis:
- ¿Tienen las estudiantes de 9vo una estatura media diferente de 175 cm?  
   H0: $\mu$ m =! 175
- ¿Es la desviación estándar de su estatura igual a o menor que 10 cm?  
   H0: $\sigma$ =< 10
- ¿Es diferente la estatura promedio de las mujeres comparada con la de los hombres?  
   H0: $\mu$ h =! $\mu$ m  
Grados de Libertad y Nivel de Significancia  
Fórmulas o Ecuaciones  
Evaluación  


### Datos

In [24]:
import pandas as pd
import matplotlib.pyplot as plt 

estaturas_grupo ={
    'nombre': ['Fer','Ale','Alex Muñoz','Gael','Erick','Angie','Karla','Angel','Jose Jacal','Leo Dorantes','Lalo','Antonio','Luis Hurtado','Cristian ','Juve',],
    'altura':[180,170,170,168,165,168,170,175,175,171,172,170,177,170,176 ], 
    'sexo':['M','M','M','M','M','F','F','M','M','M','M','M','M','M','M',]}

indexes = ['1','2','3','4','5','6','7','8','9','10','11','12','13','14','15',]
columns = ['nombre','altura','sexo']

df_estaturas = pd.DataFrame(estaturas_grupo)
df_estaturas

nombre  altura sexo
0            Fer     180    M
1            Ale     170    M
2     Alex Muñoz     170    M
3           Gael     168    M
4          Erick     165    M
5          Angie     168    F
6          Karla     170    F
7          Angel     175    M
8     Jose Jacal     175    M
9   Leo Dorantes     171    M
10          Lalo     172    M
11       Antonio     170    M
12  Luis Hurtado     177    M
13     Cristian      170    M
14          Juve     176    M

### promedio

In [35]:
promedio_estaturas = df_estaturas['altura'].mean()
promedio_estaturas


172.23076923076923

In [36]:
df_estatura_mujeres = df_estaturas [df_estaturas['sexo'] == 'F']
media_mujeres = df_estatura_mujeres ['altura'].mean()
media_mujeres

169.0

In [37]:
df_estatura_hombres = df_estaturas [df_estaturas['sexo'] == 'M']
media_hombres = df_estatura_hombres ['altura'].mean()
media_hombres

172.23076923076923

### desviación estandar

In [39]:
desv_estandar_estaturas = df_estaturas['altura'].std()
desv_estandar_estaturas

4.003569835605647

### conclusiones

- ¿Tienen las estudiantes de 9vo una estatura media diferente de 175 cm?  
   H0: $\mu$ m =! 175  
   $\mu$ m = 172.23076923076923 por lo tanto se confirma la hipótesis nula
   
- ¿Es la desviación estándar de su estatura igual a o menor que 10 cm?  
   H0: $\sigma$ =< 10
   $\sigma$ = 4.003569835605647 por lo tanto se confirma la hipótesis nula

- ¿Es diferente la estatura promedio de las mujeres comparada con la de los hombres?  
   H0: $\mu$ h =! $\mu$ m  
   $\mu$ h = 172.23076923076923 
   $\mu$ m = 169.0
   por lo tanto se confirma la hipótesis nula

***
### ejercicio datos aeropuerto

De 3 variables diferentes de la BD utilizada para el examen del primer parcial, plantear 3 hipótesis, y generar los parámetros estadísticos que sean necesarios para validar las hipótesis nulas, con los estadísticos de prueba que veremos la próxima clase.


